_data/ArpitGoyal/ArpitGoyal.md

```diff
@@ -0,0 +1,5 @@
+name: Arpit Goyal

```

Comment:
It seems like you have removed the table dashes
```
---
YOUR INFO
---
```

Reply:
Issue is resolved now. Please have a look